### DISCUSSION TITLE
Microphone mute

### DISCUSSION THREAD

**Discussion by kematian05** [seerge/g-helper#1709] [General]
in some applications muted/unmuted message didn't show. How can i fix this?

**Comment by seerge** [comment]
@VusatOrujov in older games that run in dedicated fullscreen (and use DirectX 11 or before) windows is not allowing showing anything on top of it. So there you won't see a muted/unmuted toast message.

**Reply by kematian05** [reply]
Thank you

**Reply by RAGE-HERO7** [reply]
Oh it is called as 'Toast message' not pop up message...i didn't knew that was the term for it😅
